impl/src/test/java/org/ehcache/core/events/CacheManagerListenerInteractionsTest.java

```diff
@@ -29,8 +29,8 @@
 import static org.ehcache.config.builders.ResourcePoolsBuilder.newResourcePoolsBuilder;
 import static org.hamcrest.Matchers.is;
 import static org.junit.Assert.assertThat;
-import static org.mockito.Matchers.any;
-import static org.mockito.Matchers.eq;
+import static org.mockito.ArgumentMatchers.any;
```

Comment:
The two usages of `any` in the class should both `any(Cache.class)` to verify the listener is actually handed a cache.